python_modules/dagster/dagster/_core/snap/node.py

```diff
@@ -286,6 +279,10 @@ def __new__(
             ),
         )
 
+    @property
+    def tags(self) -> Mapping[str, object]:
+        return self.tags
```

Comment:
rm?

Reply:
ah yeah that was just a trick to find references